ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml

```diff
@@ -0,0 +1,54 @@
+
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+resources:
+  '*':
+    limits:
+      memory: 3Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+promotion:
+  namespace: codeready-toolchain
+  name: registration-service-v0.1
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/codeready-toolchain/registration-service/build/_output/bin/registration-service
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: registration-service
+    
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+
+binary_build_commands: make build
+
+tag_specification:
+  name: "4.2"
+  namespace: ocp
+
+tests:
+- as: build
+  commands: make build
+  container:
+    from: src
+- as: lint
+  commands: make lint-yaml
+  container:
+    from: src
+- as: test
+  commands: make test #-with-coverage upload-codecov-report
+  container:
+    from: src
+# - as: e2e
+#   commands: make test-e2e
+#   openshift_installer_src:
+#     cluster_profile: aws
```

Comment:
@dipak-pawar can we enable this now ??  does this required to have `make test-e2e` target available in registration_service first ??  As of now `make test-e2e` is not available with reg_service.

Reply:
we can enable this, and later we can add required things for e2e on registration service side.

Reply:
@dipak-pawar thanks, enabling all target (whatever there for toolchain-operator pr) and later will add required target in reg_service, commit https://github.com/openshift/release/pull/5539/commits/8b96c66a5b4553f34679a9e3f50a30df1c9151f9